src/docs/common/snippets/common-graal-with-plugins-multi.adoc

```diff
@@ -2,13 +2,13 @@
 
 == Generate Micronaut Application Native Executables with GraalVM
 
-We will use https://www.graalvm.org/[GraalVM], the polyglot embeddable virtual machine, to generate Native executables of our Micronaut applications.
+We will use https://www.graalvm.org/[GraalVM], an advanced JDK with ahead-of-time Native Image compilation, to generate a native executable of this Micronaut application.
 
-Compiling native executables ahead-of-time with GraalVM improves startup time and reduces the memory footprint of JVM-based applications.
+Compiling Micronaut applications ahead of time with GraalVM significantly improves startup time and reduces the memory footprint of JVM-based applications.
 
 NOTE: Only Java and Kotlin projects support using GraalVM's `native-image` tool. Groovy relies heavily on reflection, which is only partially supported by GraalVM.
 
-=== Native Executable generation
+=== Native Executable Generation
```

Comment:
I also reviewed GraalVM related content. Some titles are capitalized, some not, so I corrected those too.